ctru-rs/src/applets/mii_selector.rs

```diff
@@ -56,14 +56,20 @@ pub struct MiiSelector {
 #[derive(Clone, Debug)]
 pub struct SelectionResult {
     pub mii_data: MiiData,
+
+    #[deprecated]
     pub is_mii_selected: bool,
     pub mii_type: MiiType,
 }
 
 /// Error type for the Mii selector
 #[derive(Copy, Clone, Debug, Eq, PartialEq)]
 pub enum LaunchError {
+    /// The selected Mii's data is corrupt in some way
     InvalidChecksum,
+
```

Comment:
Remove the empty space between variants. Even with doc comments we don't put newlines in between.